packages/notification-spa/package.json

```diff
@@ -12,45 +12,45 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "9.1.9",
-    "@angular/common": "9.1.9",
-    "@angular/compiler": "9.1.9",
-    "@angular/core": "9.1.9",
-    "@angular/forms": "9.1.9",
-    "@angular/platform-browser": "9.1.9",
-    "@angular/platform-browser-dynamic": "9.1.9",
-    "@angular/router": "9.1.9",
-    "@patternfly/pfe-accordion": "1.0.0-prerelease.49",
-    "@patternfly/pfe-select": "1.0.0-prerelease.49",
-    "@patternfly/pfe-toast": "1.0.0-prerelease.49",
-    "@types/jest": "25.2.3",
+    "@angular/animations": "9.1.3",
+    "@angular/common": "9.1.3",
+    "@angular/compiler": "9.1.3",
+    "@angular/core": "9.1.3",
+    "@angular/forms": "9.1.3",
+    "@angular/platform-browser": "9.1.3",
+    "@angular/platform-browser-dynamic": "9.1.3",
+    "@angular/router": "9.1.3",
+    "@patternfly/pfe-accordion": "1.0.0-prerelease.40",
+    "@patternfly/pfe-select": "1.0.0-prerelease.42",
+    "@patternfly/pfe-toast": "1.0.0-prerelease.40",
+    "@types/jest": "25.2.1",
     "apollo-angular": "1.9.1",
     "apollo-angular-link-http": "1.10.0",
-    "apollo-cache-inmemory": "1.6.6",
-    "apollo-client": "2.6.10",
-    "apollo-link": "1.2.14",
+    "apollo-cache-inmemory": "1.6.0",
+    "apollo-client": "2.6.0",
+    "apollo-link": "1.2.11",
     "graphql": "14.6.0",
-    "graphql-tag": "2.10.3",
-    "jest": "26.0.1",
-    "jest-preset-angular": "8.2.0",
-    "rxjs": "6.5.5",
-    "ts-jest": "26.1.0",
-    "tslib": "2.0.0",
-    "zone.js": "0.10.3"
+    "graphql-tag": "2.10.0",
+    "jest": "25.4.0",
+    "jest-preset-angular": "8.1.3",
+    "rxjs": "6.5.4",
+    "ts-jest": "25.4.0",
+    "tslib": "1.10.0",
+    "zone.js": "0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.901.7",
-    "@angular/cli": "9.1.7",
-    "@angular/compiler-cli": "9.1.9",
-    "@angular/language-service": "9.1.9",
-    "@types/jasminewd2": "2.0.8",
-    "@types/node": "12.12.42",
-    "codelyzer": "5.2.2",
+    "@angular-devkit/build-angular": "0.900.5",
+    "@angular/cli": "9.0.5",
+    "@angular/compiler-cli": "9.1.3",
+    "@angular/language-service": "9.1.3",
+    "@types/jasminewd2": "2.0.3",
+    "@types/node": "12.11.1",
+    "codelyzer": "5.1.2",
     "jasmine-core": "3.5.0",
-    "jasmine-spec-reporter": "5.0.2",
-    "protractor": "7.0.0",
-    "ts-node": "8.10.2",
-    "tslint": "5.20.1",
-    "typescript": "3.9.3"
+    "jasmine-spec-reporter": "4.2.1",
+    "protractor": "5.4.3",
+    "ts-node": "8.3.0",
+    "tslint": "5.18.0",
+    "typescript": "3.7.5"
```

Comment:
Why most of the packages are downgraded?

Reply:
Removed the unwanted package downgrades. And changed the verison for typescript to 8.3.8 as angular only support ts < 3.9.0

<img width="1280" alt="Screenshot 2020-06-04 at 7 42 18 PM" src="https://user-images.githubusercontent.com/12994292/83767678-8fbea880-a69b-11ea-926c-ae14dcaabc74.png">

Reply:
Okay i understand the dependency of the typescript. But make sure that typescript should be at most updated version like as i saw in message 3.9.0

Reply:
It shows less than 3.9.0. I did try installing 3.9.0 it still throws an error